server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts

```diff
@@ -43,16 +43,16 @@ investigationInfo.get('/staticInfo', (request: Request, response: Response) => {
 })
 
 investigationInfo.post('/updateInvestigationStatus', (request: Request, response: Response) => {
-    const { epidemiologyNumber, investigationStatus } = request.body;
+    const { epidemiologyNumber, investigationMainStatus, investigationSubStatus } = request.body;
```

Comment:
epidemiologyNumber could be taken from response.locals like in line 53, no?

Reply:
yes